### DISCUSSION TITLE
[@types/react-calendar] values for selected range can be null

### DISCUSSION THREAD

**Discussion by xelaz** [DefinitelyTyped/DefinitelyTyped#59702] [Issues with a @types package]
Now CalendarProps has property value definition like this:
```
value?: Date | Date[] | null | undefined;
```

But i can set on select range functionality values like this [Date, null] or [null, null] or [Date, Date] [null, Date]
Is it better to rewrite the rule in this one:
```
value?: Date | null | [Date | null, Date | null] | undefined;
```

[a comment by an automated account is not shown]

**Comment by nebarf** [comment]
I just opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/60039 changing the type as suggested by @xelaz
